cli/cmd/helm/helmInstall.go

```diff
@@ -104,6 +122,27 @@ func helmInstall(cmd *cobra.Command, args []string) error {
 
 func (c *installSpec) installStack() error {
 	var err error
+
+	// Note do not change the below order the --set flag is set
```

Comment:
I'm not sure what order this is referring to anymore

Reply:
The comment line is out dated.